### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by amar-c8y**
widget installation steps added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This Asset Properties Widget is the Cumulocity module federation plugin created
 
 | APPLICATION BUILDER | CUMULOCITY  | ASSET PROPERTIES WIDGET PLUGIN |
 | ------------------- | ----------- | ------------------------------ |
-| 2.0.x               | >= 1018.0.0 | 1.0.1                          |
+| 2.0.x               | >= 1018.0.0 | 1.0.0                          |
```
Comment (ghost) [inline README.md]: What is application builder? Why is this added here?

File: README.md [hunk 2]
```diff
@@ -22,9 +22,12 @@ Cumulocity c8ycli >=1018.0.144
 
 ## Installation
 
-### Runtime Widget Deployment?
+### To install the widget-plugin in the Custom Cockpit application, follow the steps below:
 
-- This plugin support runtime deployment. Download and install via Administrations --> Ecosystems --> Applications --> Packages.
+1.	Click on Add extension Packages button in Administration > Ecosystem > Extension, where you can add your widget zip file.
```
Comment (ghost) [inline README.md]: Always make sure to provide exact page and button names. Here for example, the page is Extensions and not Extension, the case of letters in the button Add extension package. What is the val
```suggestion
1.  Navigate to Administration > Ecosystem > Extensions
2.  Click  **Add extension package** to upload the zip file

```
What should be the extension availability?
image

File: README.md [hunk 3]
```diff
@@ -22,9 +22,12 @@ Cumulocity c8ycli >=1018.0.144
 
 ## Installation
 
-### Runtime Widget Deployment?
+### To install the widget-plugin in the Custom Cockpit application, follow the steps below:
```
Comment (ghost) [inline README.md]: A plugin can be installed on any custom application with cumulocity dashboard. So we don't have to mention custom Cockpit application.
```suggestion
To install the Asset properties 2.0 widget plugin, you must add the plugin package to the extensions first and then install it on the desired custom application.

#### To add the plugin package to extensions
```

File: README.md [hunk 4]
```diff
@@ -22,9 +22,12 @@ Cumulocity c8ycli >=1018.0.144
 
 ## Installation
```
Comment (ghost) [inline README.md:23]: Add a section for downloading the release .zip file as discussed in the call.

File: README.md [hunk 5]
```diff
@@ -22,9 +22,12 @@ Cumulocity c8ycli >=1018.0.144
...
 
-### Runtime Widget Deployment?
+### To install the widget-plugin in the Custom Cockpit application, follow the steps below:
 
-- This plugin support runtime deployment. Download and install via Administrations --> Ecosystems --> Applications --> Packages.
+1.	Click on Add extension Packages button in Administration > Ecosystem > Extension, where you can add your widget zip file.
+2.	If you already have a custom Cockpit application, navigate to its Details page (Administration > Ecosystem > Applications > Cockpit(Custom)) and then to the Plugins tab. Install the Asset properties widget plugin.  [Extensions](https://cumulocity.com/guides/users-guide/administration/#extensions).
```
Comment (ghost) [inline README.md]:
```suggestion
#### To install the widget plugin on a custom application
1. Navigate Adminstration > Ecosystem > Applications
2. Click application name to view the application details.
3. Switch to **Plugins** tab, Click **Install plugins**
4. In the resulting dialog, search for Asset properties 2.0 widget, click **Install**


To know more about extensions, see [Administration> Managing applications > Extensions](https://cumulocity.com/guides/users-guide/administration/#extensions) in *User guide*
```

File: README.md [hunk 6]
```diff
@@ -22,9 +22,12 @@ Cumulocity c8ycli >=1018.0.144
...
+3.	If you don’t have your Custom Cockpit application, navigate to Administration > Ecosystem > Applications and click Add application. In the resulting dialog, select the option Duplicate existing application. From the list of applications select Cockpit (Subscribed). Use the default values and proceed once its done follow the **2nd steps**.
+4. Now your widget is available in your Cockpit application. Navigate to the dashboard where the newly added widget is available in the list of widgets to add.
```
Comment (ghost) [inline README.md]: Remove these lines and add a note instead
```suggestion
Note: You need a custom application with Cumulocity IoT dashboard to see the widget in action. A quick way to achieve this is by creating a custom Cockpit application.
To create  a custom Cockpit application, follow the steps below:
1. Navigate to Administration > Ecosystem > Applications 
2. click **Add application**. 
3. In the resulting dialog, select the option **Duplicate existing application**. 
4. From the list of applications, select Cockpit (Subscribed). Use the default values and proceed.
```

**Review by ghost — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (amar-c8y) [reply README.md]: extension availability means the allpication availability for sub tenant, you will get more info click on question mark icon.

File: README.md (hunk 4 above)
Comment (amar-c8y) [reply README.md:23]: yes, once release is done I need to provide release path to download the zip

File: README.md (hunk 1 above)
Comment (amar-c8y) [reply README.md]: incorporated

File: README.md (hunk 4 above)
Comment (amar-c8y) [reply README.md:23]: release path added.

File: README.md [hunk 7]
```diff
@@ -1,12 +1,12 @@
-# Cumulocity Asset Properties Widget Plugin
+# Cumulocity Asset Properties Widget Plugin [<img width="35" src="https://user-images.githubusercontent.com/32765455/211497905-561e9197-18b9-43d5-a023-071d3635f4eb.png"/>](https://github.com/SoftwareAG/cumulocity-asset-properties-widget/releases/download/sag-pkg-asset-properties-widget-1.0.0/sag-pkg-asset-properties-widget-1.0.0.zip)
 
 This Asset Properties Widget is the Cumulocity module federation plugin created using c8ycli. This plugin can be used in Application Builder or Cockpit. It fetches Inventory data based on the Asset and displays the same in a widget.
 
-### Please choose Asset Properties Widget release based on Cumulocity/Application builder version:
+### Compatible:
```
Comment (ghost) [inline README.md]:
```suggestion
### Compatibility:
```

File: README.md [hunk 8]
```diff
@@ -22,9 +22,28 @@ Cumulocity c8ycli >=1018.0.144
...
+
+#### To install the widget plugin on a custom application
+
+1. Navigate Adminstration > Ecosystem > Applications
+2. Click application name to view the application details.
+3. Switch to **Plugins** tab, Click **Install plugins**
+4. In the resulting dialog, search for Asset properties 2.0 widget, click **Install**
+
+To know more about extensions, see [Administration> Managing applications > Extensions](https://cumulocity.com/guides/users-guide/administration/#extensions) in _User guide_
+Note: You need a custom application with Cumulocity IoT dashboard to see the widget in action. A quick way to achieve this is by creating a custom Cockpit application.
```
Comment (ghost) [inline README.md:42]:
```suggestion


Note: You need a custom application with Cumulocity IoT dashboard to see the widget in action. A quick way to achieve this is by creating a custom Cockpit application.
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Review by spau6 — APPROVED** [review]

**Pull request merged**